### PR TITLE
feat: I can upload a photo on person show page

### LINKED ISSUE #1022: I can upload a photo on person show page
We recently added base64 images to the profiles that are initially seeded but it's a bit misleading because as of today you cannot upload a photo to a contact. 

We should be able to click on the avatar circle. That will open a window to upload an image from my local drive.
See how it's done in settings > profile for inspiration on picture upload.

Screenshot 2023-07-31 at 16 31 46

```[tasklist]
### Tasks
- [ ] Right now, we have one ShowPageSummaryCard component for companies and people. Their behavior is too different, we should create two separate components in @/companies and @/people
- [ ] In the People one, we can add a onclick function that triggers the upload
- [ ] We need to create a new resolver on people to upload a picture. You can take profile picture upload as an example
- [ ] Make sure to check ability-handler: you should not be able to update a person out of your workspace
- [ ] You will also need to introduce a new FileFolder
```

### PR DESCRIPTION
**Pull request by gitstart-twenty**
fixes #1022

### CODE REVIEW

**Comment by gitstart-twenty** [comment]
Hey @charlesBochet @FelixMalfait,

Should we consider supporting users being able to remove the avatar or we just need to care about the avatar uploads for now?

**Comment by charlesBochet** [comment]
> Hey @charlesBochet @FelixMalfait,
> 
> Should we consider supporting users being able to remove the avatar or we just need to care about the avatar uploads for now?

Good point, as a v1, we don't want to be able to remove. But re-clicking the image should allow to replace the image

File: server/src/core/person/person.resolver.ts [hunk 1]
```diff
@@ -156,4 +164,31 @@ export class PersonResolver {
       select: prismaSelect.value,
     } as Prisma.PersonCreateArgs);
   }
+
+  @Mutation(() => String)
+  async uploadPersonPicture(
+    @Args('id') id: string,
```
Comment (charlesBochet) [inline server/src/core/person/person.resolver.ts:170]: let's add permission check here too
  @UseGuards(AbilityGuard)
  @CheckAbilities(UpdatePersonAbilityHandler)

Comment (charlesBochet) [reply server/src/core/person/person.resolver.ts:170]: And check that we cannot modify the picture of a person out of the user's workspace

**Review by charlesBochet — COMMENTED** [review]
The approach looks good!

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**